src/Core/src/Platform/Tizen/WindowExtensions.cs

```diff
@@ -110,5 +111,14 @@ static void OnBackButtonPressed(Window window)
 				s_windowCloseRequestHandler[window].Invoke();
 		}
 
+		internal static Devices.DisplayOrientation GetOrientation(this IWindow window) =>
+			window.Handler?.MauiContext?.GetPlatformWindow()?.Rotation switch
+			{
+				0 => Devices.DisplayOrientation.Portrait,
+				90 => Devices.DisplayOrientation.Landscape,
+				180 => Devices.DisplayOrientation.Portrait,
+				270 => Devices.DisplayOrientation.Landscape,
+				_ => Devices.DisplayOrientation.Unknown
+			};
```

Comment:
On second thought, we should consider devices that are basically in landscape mode, such as TVs. So I think it would be better to modify it as below.
```suggestion
		internal static Devices.DisplayOrientation GetOrientation(this IWindow window)
		{
			bool isTV = Elementary.GetProfile() == "tv";
			return window.Handler?.MauiContext?.GetPlatformWindow()?.Rotation switch
			{
				0 => isTV ? Devices.DisplayOrientation.Landscape : Devices.DisplayOrientation.Portrait,
				90 => isTV ? Devices.DisplayOrientation.Portrait : Devices.DisplayOrientation.Landscape,
				180 => isTV ? Devices.DisplayOrientation.Landscape : Devices.DisplayOrientation.Portrait,
				270 => isTV ? Devices.DisplayOrientation.Portrait : Devices.DisplayOrientation.Landscape,
				_ => Devices.DisplayOrientation.Unknown
			};
		}
```